backend/inspirehep/matcher/api.py

```diff
@@ -179,3 +179,20 @@ def match_reference_control_numbers(reference):
         list: list of `control_numbers` or None.
     """
     return match_references(reference)
+
+
+def match_references_next(references):
```

Comment:
```suggestion
def match_references(references):
```

and rename https://github.com/inspirehep/inspirehep/pull/1224/files#diff-53c058a59677fd40d2e3132ba2e75e88R145 to ``match_reference_config``